content/docs/get-started-with-neon/using-api-keys.md

```diff
@@ -12,8 +12,94 @@ Once authenticated, a user can create and access Projects and [query Project dat
 
 ## Using API keys as a Neon User
 
-API keys allow users to access Neon application programming interface.
+API keys allow users to access the Neon application programming interface.
 
-An API key provides access to any action available to the user. Currently, API keys cannot be scoped to the specific Projects. Neon users can provision multiple API keys. An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked.
+An API key provides access to any action available to the user.An API key that is no longer needed can be revoked; this action cannot be reverted. Any issued API key is valid forever until it is revoked. Neon users can generate multiple API keys.
+
+### Issue a New API Key
+
+Here's how to issue a new API key:
+
+1. Start by logging in to [Neon Console](https://console.neon.tech).
+2. Click on your username in the upper right corner, then click the `Settings` button.
```

Comment:
It's not the Settings, but Account.
Additionally, not sure if we should call this a 'button'.
<img width="349" alt="Screenshot 2022-08-02 at 18 53 44" src="https://user-images.githubusercontent.com/445400/182431176-ed741169-f25f-4ccf-9115-92d9bf9b6b4b.png">

Reply:
@WeaponXII Did you address this comment and delete the word `button`?

Reply:
@pkafei yes, my last commit resolved this

Reply:
Thanks!